src/constants/label.js

```diff
@@ -1,9 +1,24 @@
 "use strict";
 
 // Name fields in order of preference
-export const name_en = [
-  "coalesce",
-  ["get", "name:en"],
-  ["get", "name:latin"],
-  ["get", "name"],
-];
+export const localizedName = (function () {
```

Comment:
The `text-field` expressions continue to be evaluated statically as the page loads. To dynamically switch languages after the page loads, we could listen for the [`languagechange`](https://developer.mozilla.org/en-US/docs/Web/API/Window/languagechange_event) and tear down and rebuild all the layers. This would require refactoring americana.js.

Reply:
#579